### PR TITLE
Gfx view

### PR DESCRIPTION
**Pull request by kret0**
implemented window view.
I knew we need to refactor this but also we want some insight to our simulation right now.

### CODE REVIEW

**Comment by kret0** [comment]
Please, take a look and suggest some changes btw. Right now ik we need to make terminal view also runnable and change interface of view as well as refactor simulation.

File: src/main/java/com/codecool/view/windowedview/WindowHandler.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package com.codecool.view.windowedview;
+
+import com.codecool.view.windowedview.display.Camera;
+import com.codecool.view.windowedview.input.KeyManager;
+
+public class WindowHandler {
+
+    private Camera camera = new Camera(this, 0,0);
+    private KeyManager keyManager = new KeyManager();
+    private int width=1800,height=800;
```
Comment (idzia) [inline src/main/java/com/codecool/view/windowedview/WindowHandler.java:10]: it shouldn't be hardcoded

File: src/main/java/com/codecool/view/windowedview/WindowRunnable.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+        this.width = width;
+        this.height = height;
+        this.title = title;
+        this.board = board;
+
+        keyManager = new KeyManager();
+    }
+
+    private void init(){
+        display = new Display(title, width, height);
```
Comment (idzia) [inline src/main/java/com/codecool/view/windowedview/WindowRunnable.java:45]: it should get board width and height as well

File: src/main/java/com/codecool/view/windowedview/WindowRunnable.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+        if(bs == null){
+            display.getCanvas().createBufferStrategy(3);
+            return;
+        }
+        g = bs.getDrawGraphics();
+        //Clear Screen
+        g.clearRect(0, 0, width, height);
+        //Draw Here!
+        tiles.render(g);
+
```
Comment (idzia) [inline src/main/java/com/codecool/view/windowedview/WindowRunnable.java:72]: implement camera view

File: src/main/java/com/codecool/view/windowedview/WindowedView.java [hunk 4]
```diff
@@ -2,9 +2,19 @@
...
+public class WindowedView implements View  {
+
+    WindowRunnable runnable;
+
+    public WindowedView(Board board){
+        Assets.init();
+        runnable = new WindowRunnable(board,"Life-Simulator", 1800,700);
+
+    }
     public void draw(Board board) {
```
Comment (idzia) [inline src/main/java/com/codecool/view/windowedview/WindowedView.java:16]: ?

**Review by idzia — COMMENTED** [review]
should be refactored later

**Pull request merged**